src/main/resources/assets/securitycraft/lang/en_us.json

```diff
@@ -498,17 +498,16 @@
 	"gui.securitycraft:scManual.viewActivatedBlock": "View activated: This block can be activated by hovering over it with your cursor while in block-breaking range.",
 	"gui.securitycraft:scManual.explosiveBlock": "Explosive: This block is able to be remotely detonated, and may be able to be activated/deactivated, using the Mine Remote Access Tool.",
 	"gui.securitycraft:scManual.customizableBlock": "Customizable: You may use the Universal Block Modifier and the various modules added by SecurityCraft to customize the functions of this block.",
-    "gui.securitycraft:scManual.options": "Available options:",
-    "gui.securitycraft:scManual.modules": "Available modules:",
+    	"gui.securitycraft:scManual.options": "Available options:",
+    	"gui.securitycraft:scManual.modules": "Available modules:",
```

Comment:
Unnecessary change. Please revert the line to how it was before.

Reply:
Raqbit reeee’d me for not having indentations here, so just checking you’re aware that this is a format change, not a content change.

Reply:
Raqbit reeee’d at me for not having these lines indented, so just verifying with you that you know this is a format/cosmetic change and not a content change. Will still revert if you want, of course.

Reply:
Yes, I am aware. However you added a tab instead of replacing the 4 spaces with it.  I'll fix indentation myself in these files.